[CrowdStrikeFalcon] new function: "safe_strptime" (#41384)
demisto/content: Packs/CrowdStrikeFalcon/ReleaseNotes/2_3_12.md

#### Integrations

##### CrowdStrike Falcon

> - Fixed an issue where the integration would throw an error when parsing timestamps without microsecond components.

Reviewer: ```suggestion
- Fixed an issue with the timestamp parsing logic to prevent errors when microsecond components are missing.
```
